src/hive/services/directions.cljs

```diff
@@ -1,41 +1,34 @@
 (ns hive.services.directions
   (:require [clojure.string :as str]
-            [cljs.spec.alpha :as s]))
+            [cljs.spec.alpha :as s]
+            [hiposfer.geojson.specs :as geojson]))
 
-(s/def ::input (s/and string? not-empty))
-(s/def ::coordinates (s/tuple number? number?))
-(s/def ::alternatives boolean?)
-(s/def ::geometries #{"geojson" "polyline" "polyline6"})
-(s/def ::overview #{"simplified" "full" "false"})
+;(s/def ::coordinates (s/tuple number? number?))
+;(s/def ::alternatives boolean?)
+;(s/def ::geometries #{"geojson" "polyline" "polyline6"})
+;(s/def ::overview #{"simplified" "full" "false"})
```

Comment:
Is it possible to share the specs with kamal?

Reply:
Yes it is possible. However we would obviously have to make those specs an independent library, like I did with `geojson.specs`

Reply:
Yes exactly!